### PR TITLE
Change the way Service Agents are accessed in swagger

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

Related to https://github.com/transcom/mymove/pull/870

~~Unfortunately the `SwaggerField` we use for forms can't do nested definitions.  Also, to be more straight forward in coding the backend and for api calls I've just flattened out the ShippingAgent definition.~~

I've created a new model in Related to https://github.com/transcom/mymove/pull/870.  Based on some early SBD wireframes it made sense to treat the accepting of a shipment separate from managing the service agents related to that shipment.  We can still make acceptance rely on having Service Agents but the API will expose differently.  Now I have these endpoints:

- /shipment/{shipment_uuid}/accept (POST)
- /shipment/{shipment_uuid}/service_agents/ (GET, POST)
- /shipment/{shipment_uuid}/service_agents/{service_agent_uuid} (PATCH, DELETE)

Additionally I fixed a typo issue with the ShipmentStatus enum.

## Code Review Verification Steps

* [x] Request review from a member of a different team.

### CODE REVIEW

**Comment by chrisgilmerproj** [comment]
@tinyels - that's good to know.  I'm still wondering if this might not be an easier format to use the API as well.  Before you had to pass in two objects with three fields each.  Now you just submit one object with six fields.  I'm thinking it might be nicer from a UX perspective.  What do you think?

@amitch23 - I'd still love to talk to you about how nested definitions work with `SwaggerField`.

**Comment by chrisgilmerproj** [comment]
I learned Friday about a different, more generic way that SBD has been thinking about this.  I'll be updating this again shortly.  Here's a screen shot I got of their ideas:

screen shot 2018-08-10 at 2 07 11 pm

**Comment by chrisgilmerproj** [comment]
@tinyels - will you take a look at this?  I've basically rethought how we would manage these contacts now and I think it makes more sense.

File: swagger/api.yaml [hunk 1]
```diff
@@ -210,7 +210,7 @@ definitions:
         type: string
         format: uuid
         example: c56a4180-65aa-42ec-a945-5fd21dec0538
-      shipment_id:
+      shipment_uuid:
```
Comment (tinyels) [inline swagger/api.yaml]: This is locally consistent (within the api.yaml) but we have been using `*_id` in the internal yaml.

Comment (chrisgilmerproj) [reply swagger/api.yaml]: Yeah, I've been thinking of moving it all over to `_id` because I noticed the difference.  I'll make a PR for this soon.

**Pull request merged**